lib/find-available-spaces.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+const canMoveOffBoard = require("./can-move-off-board.js");
+const canMoveToSpace = require("./can-move-to-space.js");
+const _ = require("lodash");
+
+function findAvailableSpaces(gameState, numberOfSpaces) {
+    const isPlayerOne = gameState.isPlayerOne;
+    const activePlayer = isPlayerOne ?
+        {
+            playerObject: gameState.playerOne,
+            startSpace: require("./constants").PLAYER_ONE_START_SPACE,
```

Comment:
It would be cleaner to just have `const constants = require('./constants');` at the top of the file with the rest of your `require`s, and then just make this `constants.PLAYER_ONE_START_SPACE`.